code/modules/mob/living/silicon/pai/software.dm

```diff
@@ -22,6 +22,7 @@
 															"security HUD" = 20,
 															"loudness booster" = 20,
 															"newscaster" = 20,
+															"internal gps" = 20,
```

Comment:
```suggestion
															"internal gps" = 35,
```
RaveRadbury's suggestion is good, let's start conservative and tweak if necessary.